www.foia.gov/_includes/footer.html

```diff
@@ -1,25 +1,18 @@
 <!-- Start Footer -->
-<div id="footer"><a href="index.html"><img src="images/foia-footer.gif" alt="foia.gov" /><span>Foia.gov</span></a>
-  <div id="footerNav">   
-	<a id="footerLinks"></a>
-  <ul>
-    <li><a href="sitemap.html">Site Map</a></li>
-	<li><a href="https://www.justice.gov/accessibility/accessibility-information">Accessibility</a></li>
-	<li><a href="foia-info.html">FOIA</a></li>
-	<li><a href="https://www.justice.gov/doj/privacy-policy">Privacy Policy</a></li>
-	<li><a href="https://www.justice.gov/legalpolicies">Policies &amp; Disclaimers</a></li>
-	<li><a href="quality.html">Information Quality</a></li>
-	<li><a href="developer.html">Developer Resources</a></li>
-	<li><a href="http://www.justice.gov">Justice.gov</a></li>
-	<li><a href="http://www.usa.gov">USA.gov</a></li>
-  </ul>
-  </div>
-  <div class="clear">&nbsp;</div>
-  <div align="center">
-    <a href="http://www.data.gov/"><img src="images/datagov-footer.gif" alt="data.gov" /><span>Data.gov</span></a>
-	<a href="http://www.whitehouse.gov/open"><img src="images/opengov-footer.gif" alt="whitehouse.gov/open" /><span>Whitehouse.gov/open</span></a>
-	<a href="http://www.usaspending.gov/"><img src="images/usaspending-footer.gif" alt="usaspending.gov" /><span>USASpending.gov</span></a>
-    <a href="http://www.recovery.gov/"><img src="images/recoverygov-footer.gif" alt="recovery.gov" /><span>Recovery.gov</span></a>
-  </div>
+{% unless page.not_legacy %}
```

Comment:
This is a little funny with the double negative... maybe we want to swap this to `page.modern` or `page.uswds`?

Reply:
I like `page.uswds`! Will change.